revolv/payments/migrations/0031_auto_20150602_0011.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are a ridiculous amount of migration files here, and running migrations takes long enough as it is. Do you think you can delete the new ones then run a `makemigrations` again so that there's only one payments migration file merged with this PR?